jenkins/install-jenkins-and-docker-build-pipeline.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+if [ "$#" -lt 5 ]; then
+    echo "Usage: ./install-jenkins-and-docker-build-pipeline.sh <include Docker Build pipeline>"
+    echo "                                                      <git url>"
+    echo "                                                      <container registry url>"
+    echo "                                                      <container registry user name>"
+    echo "                                                      <container registry password>"
+    exit 1
+fi
+include_pipeline=${1}
+git_url=${2}
+cr_url=${3}
+cr_user=${4}
+cr_password=${5}
+
+#install jenkins
+wget -q -O - https://pkg.jenkins.io/debian/jenkins-ci.org.key | sudo apt-key add -
+sudo sh -c 'echo deb http://pkg.jenkins.io/debian-stable binary/ > /etc/apt/sources.list.d/jenkins.list'
+curl -fsSL https://apt.dockerproject.org/gpg | sudo apt-key add -
+sudo add-apt-repository \
+       "deb https://apt.dockerproject.org/repo/ \
+       ubuntu-$(lsb_release -cs) \
+       main"
+sudo apt-get update --yes
+sudo apt-get install jenkins --yes
+sudo apt-get install jenkins --yes
+sudo apt-get install git --yes
+sudo apt-get -y install docker-engine --yes
+sudo gpasswd -a jenkins docker
+sudo service docker restart
+skill -KILL -u jenkins
+sudo service jenkins restart
+
+if [[ "${include_pipeline}" != "Include" ]]
```

Comment:
If we really expect people to reuse these scripts outside of the quickstart, we should change the 'include_pipeline' to a flag or boolean. If you're calling this script directly from azuredeploy.json, I think you can change the commandToExecute to be something like this:

./test.sh $(expr "${x}" == "foo")